lighthouse-extension/gulpfile.js

```diff
@@ -174,6 +194,7 @@ gulp.task('compilejs', () => {
       file.contents = new Buffer(minified);
       return file;
     }))
+    .pipe(banner(BANNER))
```

Comment:
Since it's a build step (and so just devDependencies), I personally don't particularly mind using yet another gulp library instead of writing our own function. It seems like `gulp-header` would let you inject this comment pretty easily at the top.